javascript-practice/src/js/constants/variable.js

```diff
@@ -0,0 +1,20 @@
+export const URL = {
```

Comment:
Variable is got a good name, it so common

Reply:
Fixed.